pkg/netutil/netutil.go

```diff
@@ -42,17 +42,31 @@ type CNIEnv struct {
 	Networks    []*networkConfig
 }
 
-func NewCNIEnv(cniPath, cniConfPath string) (*CNIEnv, error) {
+type NewCNIEnvOpts func(e *CNIEnv) error
```

Comment:
Maybe change `NewCNIEnvOpts` to `NewCNIEnvOpt` since the type itself is not plural (not an array of opts) and only means one `object/opt`.

Reply:
I prefer `NewCNIEnvOpts` to indicate that this allows the caller to set `options` for the new CNIEnv.

Reply:
The plural form isn't consistent with the (de facto) standard convention